pkg/config/config.go

```diff
@@ -369,6 +369,16 @@ func loadProxyFromEnv() {
 		return os.Getenv(strings.ToLower(key))
 	}
 
+	// getEnvDDPrecedence pulls the value of the `DD_`-prefixed env var first, and
+	// if not found or empty, the value of the env var without the prefix (case-insensitive)
+	getEnvDDPrecedence := func(key string) string {
+		value := os.Getenv("DD_" + key)
+		if value == "" {
```

Comment:
if `DD_PROXY_HTTP` is set (ie: manual action from the user): I guess we should use it. This would mean not using any proxy.

We could write an info message in that case. But it might be less confusing then using `HTTP_PROXY` when you did set `DD_PROXY_HTTP`.

We could use `os.LookupEnv` for this.

Reply:
makes sense, I'll implement this